### PR TITLE
update lexical version 0.15.0

### PR DESCRIPTION
**Pull request by taishinaritomi**

### CODE REVIEW

**Comment by wobsoriano** [comment]
Thanks! Would you like to update the playground as well?

**Comment by taishinaritomi** [comment]
I have unified the versions of the playground and lexical-vue.

File: package.json [hunk 1]
```diff
@@ -55,36 +55,36 @@
     "docs:serve": "vitepress serve docs"
   },
   "peerDependencies": {
-    "lexical": "0.14.5",
-    "vue": "^3.2.0"
+    "lexical": "0.15.0",
+    "vue": "^3"
```
Comment (wobsoriano) [inline package.json:59]: Can we put this back to `^3.2.0`? I want it to have this minimum version.

I think I put 3.2.x as minimum to make `Teleport` available. I'm not sure

**Review by wobsoriano — APPROVED** [review]
I appreciate the help!

**Comment by TaishiNaritomiAPG** [comment]
v0.8.4?

**Pull request merged**